tests/test_movies.py

```diff
@@ -0,0 +1,49 @@
+# Get movie details endpoint
+# Mock: data frame with id and tmdbId
+#       get_movie_details_from_tmdb response (MoviePublic instance)
+
+from app.data import get_movie_details_from_tmdb, get_movie_df
+from app.main import app
+
+from app.data import get_movie_details_from_tmdb, get_movie_df
+from app.models.movie import MoviePublic
+
+from httpx import AsyncClient, codes
+import pytest
+import pandas as pd
+
+def mock_get_movie_df():
+  mock_data = { 'movieId': [1], 'imdbId': [321], 'tmdbId': [123] }
+  return pd.DataFrame(mock_data)
+
+def mock_get_movie_details_from_tmdb() -> MoviePublic:
+  return MoviePublic(id=123, title="Mock Title", thumbnail_url="https://example.com/thumbnail.png")
+
+@pytest.mark.asyncio
```

Comment:
I was thinking initially we should write all tests in synchronous way. But according to fastapi it makes sense to write Async tests, for example, when you're querying your database asynchronously. Imagine you want to test sending requests to your FastAPI application and then verify that your backend successfully wrote the correct data in the database, while using an async database library.

[Reference Guide](https://fastapi.tiangolo.com/id/advanced/async-tests/)